webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -272,6 +274,16 @@ function SystemUsers(props: Props) {
                                 title={getDisplayName(info.row.original)}
                             >
                                 {getDisplayName(info.row.original) || ''}
+                                {' '}
```

Comment:
Do we always want this arbitrary whitespace here? Wouldn't it be better to space the tag properly away from the display name span?

Reply:
Yes removed and handled by css